bot/exts/moderation/infraction/management.py

```diff
@@ -49,7 +49,7 @@ async def infraction_group(self, ctx: Context) -> None:
     async def infraction_append(
         self,
         ctx: Context,
-        infraction_id: t.Union[int, allowed_strings("l", "last", "recent")],  # noqa: F821
+        infraction: Infraction,  # noqa: F821
```

Comment:
This noqa isn't needed. It was here because flake8 doesn't handle function calls with strings (allowed_strings) very well.